addons/packages/antrea/1.5.2/README.md

```diff
@@ -0,0 +1,57 @@
+| `antrea.config.antreaProxy.ProxyLoadBalancerIPs` | Optional          | Load-balance traffic destined to the External IPs of LoadBalancer services. Default: false                    |
+| `antrea.config.kubeAPIServerOverride`            | Optional          | Provide the address of Kubernetes apiserver. Default: nil                                                     |
+| `antrea.config.transportInterface`               | Optional          | The name of the interface on Node which is used for tunneling or routing the traffic. Default: empty          |
+| `antrea.config.transportInterfaceCIDRs`          | Optional          | The network CIDRs of the interface on Node which is used for tunneling or routing the traffic. Default: empty |
+| `antrea.config.multicastInterfaces`              | Optional          | The names of the interfaces on Nodes that are used to forward multicast traffic. Default: nil                 |
+| `antrea.config.trafficEncryptionMode`            | Optional          | Determines how tunnel traffic is encrypted. Default: none                                                     |
+| `antrea.config.wireGuard.port`                   | Optional          | The port for WireGuard to receive traffic. Default: 51820                                                     |
+| `antrea.config.enableUsageReporting`             | Optional          | Enable usage reporting (telemetry) to VMware. Default: false                                                  |
+| `antrea.config.serviceCIDR`                      | Optional          | The service IPv4 CIDR to use. Default: `10.96.0.0/12`                                                         |
+| `antrea.config.serviceCIDRv6`                    | Optional          | The service IPv6 CIDR to use. Default: nil                                                                    |
+| `antrea.config.trafficEncapMode`                 | Optional          | The traffic encapsulation mode. Default: `encap`                                                              |
+| `antrea.config.noSNAT`                           | Optional          | Boolean flag to enable/disable SNAT. Default: `false`                                                         |
+| `antrea.config.disableUdpTunnelOffload`          | Optional          | Disable UDP tunnel offload feature on default NIC. Default: `false`                                           |
+| `antrea.config.defaultMTU`                       | Optional          | MTU to use. Default: `null` (Antrea will autodetect)                                                          |
+| `antrea.config.tlsCipherSuites`                  | Optional          | List of allowed cipher suites                                                                                 |
+| `antrea.config.featureGates.AntreaProxy`         | Optional          | Boolean flag to enable/disable antrea proxy. Default: `false`                                                 |
```

Comment:
According to values.yaml, AntreaProxy is enabled by default. It's enabled by default in previous Antrea release, too.